deps/rabbit/test/amqp_client_SUITE.erl

```diff
@@ -358,7 +359,11 @@ end_per_testcase(Testcase, Config) ->
     %% Assert that every testcase cleaned up.
```

Comment:
This comment can now be deleted. We don't assert anymore that testcases cleaned up. We just force cleanup from now on by force terminating any connection and sessions and force deleting any queues.

Reply:
fixed